src/custom-element.js

```diff
@@ -1033,38 +1033,32 @@ export function createAmpElementProto(win, name, implementationClass) {
    * @package @final
    */
   ElementProto.overflowCallback = function(overflown, requestedHeight) {
-    if (!overflown && !this.overflowElement_) {
-      // Overflow has never been initialized and not wanted.
-      return;
-    }
-
-    const overflowElement = this.getOverflowElement();
-    if (!overflowElement) {
+    this.getOverflowElement();
```

Comment:
I'd prefer if the generic element code to run first before the custom overridden code executes.